site/en/adminGuide/upgrade-helm.md

```diff
@@ -0,0 +1,85 @@
+---
+id: upgrade-helm.md
+label: Helm
+order: 1
+group: upgrade.md
+summary: Learn how to upgrade Milvus v2.1 to v2.2 with Helm.
+---
+
+{{tab}}
+
+# Upgrade Milvus with Helm
+A major change in Milvus 2.2 is the meta structure of segment indexes. This topic describes how to use Helm to migrate the meta and upgrade Milvus from v2.1.x to v2.2.0.
+
+<div class="alert note">
```

Comment:
Here we don't provide any information for the meta migration script. We'd better explain that we provide a script to help user to migrate meta.